TestModels/aws-sdks/sqs/smithy-build.json

```diff
@@ -1,5 +1,23 @@
 {
   "version": "1.0",
+  "projections": {
```

Comment:
Could we make this `listQueuesOnly` and actually hook it up as the source in the Gradle build via `projectionName = "listQueuesOnly"` as above? That way we're actually testing the workflow we're claiming to work now, and `TestComAmazonawsSqs.dfy` should still work AFAICT.

Reply:
Yeah, good call!

Reply:
Thanks! Only nit is that "listQueuesOnly" is now technically "listQueuesOnlyExceptForAtLeastOneOtherOneBecauseOf#439" :) But it doesn't look like `smithy-build.json` supports comments. Perhaps we could add a comment to the README of this test model?

Reply:
good point, I'll do both. 
(JSON in general doesn't support comments, it's kind of a bummer)